alfresco-webscripts-client/alfresco-webscripts-ditto/build.gradle

```diff
@@ -0,0 +1,78 @@
+plugins {
+    id 'java-library'
+    id 'maven-publish'
+}
+
+group = rootProject.group
+version = rootProject.version
+
+repositories {
+    mavenLocal()
+    mavenCentral()
+    maven { url 'https://artifacts.alfresco.com/nexus/content/groups/public/' }
+
+    repositories {
+        maven {
+            name 'artifactory.xenit.eu'
```

Comment:
This is a public project. It should not depend on our internal artifactory.

Reply:
Fair point, but I have not figured out how to publish snapshots to sonatype yet

Reply:
ditto should be published on sonatype first ?